src/mjolnir/landmark_database_builder.cc

```diff
@@ -1,8 +1,39 @@
 #include "mjolnir/landmark_database_builder.h"
 #include "mjolnir/util.h"
 
+#include "mjolnir/osmpbfparser.h"
+
 namespace valhalla {
 namespace mjolnir {
+const std::string landmark_db = "landmarks.db";
+
+static const std::unordered_map<std::string, LandmarkType> str_type_map = {
+    {"parking", LandmarkType::parking},
+    {"bench", LandmarkType::bench},
+    {"parking_space", LandmarkType::parking_space},
```

Comment:
i dont think any of these make sense really. maybe the first one whic could be a parking lot, but in reality it could but just some spaces on the side of the road. remember we are trying to call out things that are important, things people wont have to think about to see to navigate by

Reply:
oh and i should say by any i mean the top 3 here. if you noticed github lets you mark multiple lines but it doesnt actually highlight them when viewing the commment, but see above how it says +11 to +13. im not sure if you were aware of this github gotcha :smile:

Reply:
yes I can see the highlighted 3 lines. 
For the landmrks here I just chose the 15 most popular ones. it make sense to carefully choose some obvious and straightfoward things. but from the amenity website most of the obvious landmarks are not very popular. we dont consider that as an issue right now?

Reply:
correct: some of the obvious landmarks are still popular like "restaurant". so popularity is not an issue :)